src/pygeoprocessing/routing/routing.pyx

```diff
@@ -4190,7 +4241,9 @@ def detect_outlets(d8_flow_dir_raster_path_band, target_outlet_vector_path):
                     outlet_feature = None
                     outlet_point = None
 
-    LOGGER.info('outlet detection: committing transaction')
+    d8_flow_dir_raster = None
+    d8_flow_dir_band = None
+    LOGGER.info('outlet detection: 100% complete -- committing transaction')

```

Comment:
I like that you added the logging at 0 and 100% completion - it can be confusing having the 5 second logging intervals if it never indicates it actually finished.

Reply:
Ahh that's nice you noticed / bad that you noticed when it's not there. I agree that's annoying and I try to fix it whenever I get the chance.